Assets/MRTK/StandardAssets/Shaders/MixedRealityTextMeshProSprite.shader

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// NOTE: MRTK Shaders are versioned via the MRTK.Shaders.sentinel file.
+// When making changes to any shader's source file, the value in the sentinel _must_ be incremented.
+
+// Text Mesh Pro Sprite shader with MRTK Additions

```

Comment:
Is this copying code from Unity's shader? If so, what's the licensing on that?

Reply:
This is copying code from: [Unity TMP Sprite shader](https://github.com/UnityTechnologies/Presentation/blob/master/TextMesh%20Pro/Resources/Shaders/TMP_Sprite.shader ) and [MRTK TMP shader](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/main/Assets/MRTK/StandardAssets/Shaders/MixedRealityTextMeshPro.shader)

The MRTK TMP shader has a similar header and it's code was copied from the same Unity repository from the looks of it.

Reply:
Did some digging and that project doesn't seem to have a license. Other project under the UnityTechnoloigies boarder project on github have used the apache license, so I think we should be good here.

https://github.com/Unity-Technologies/com.unity.perception/blob/master/com.unity.perception/LICENSE.md

Reply:
@RogPodge I'm not sure if this one is Apache. According to the package documentation at https://docs.unity3d.com/Packages/com.unity.textmeshpro@3.2/license/LICENSE.html, it's licensed under the Unity Companion License for Unity-dependent projects.

https://unity3d.com/legal/licenses/Unity_Companion_License?_ga=2.57595874.953248416.1630340595-862490274.1597860099

Either way though, we should ensure this is registered in our cgmanifest and makes sure distributed a derived shader is in compliance.

Reply:
@keveleigh I updated the cgmanifest and the license in the header of the TMP derived shaders in our repo with the linked Unity Companion License.